test/__snapshots__/Button.test.tsx.snap

```diff
@@ -2,15 +2,15 @@
 
 exports[`Button add classes 1`] = `
 <button
-  className="button button-primary"
+  className="button button-primary size--medium"
```

Comment:
Ok... Could you tell me a connection between to msg in a commit and the changes?